Source/WebCore/page/Page.cpp

```diff
@@ -2333,6 +2333,10 @@ void Page::doAfterUpdateRendering()
         document.enqueuePaintTimingEntryIfNeeded();
     });
 
+    forEachDocument([] (Document& document) {
```

Comment:
This should really be using `forEachRenderableDocument`. The spec refers to the `docs` variable which is filtered before hand in the HTML standard.